### DISCUSSION TITLE
Needed package xlhtml

### DISCUSSION THREAD

**Discussion by Lasaise** [jsuto/piler#113] [General]
Hello,

in the Installation Documentation (https://www.mailpiler.org/installation/) are some highly recommend pakages like xlhtml with sj mod.
In the old BitBucket Repo was a file for this package that you could download.
Is this package still needed on a Debain 12 with mailpiler 1.4.6 Installation?

Can anyone provide me the file ? This is the old link for teh file
https://bitbucket.org/jsuto/piler/downloads/xlhtml-0.5.1-sj-mod.tar.gz

**Comment by jsuto** [comment]
Hello. Well, not sure if you need it. It allows you to extract textual content from xls files. Since nowadays xlsx (or perhaps ods) is the dominant format, you may be fine without it.

**Comment by jsuto** [comment]
I've uploaded it to https://github.com/jsuto/piler/releases/download/piler-1.4.6/xlhtml-0.5.1-sj-mod.tar.gz until I find a better place.
